src/common/actions/repositories.js

```diff
@@ -1,66 +1,29 @@
-import 'isomorphic-fetch';
-
-import { checkStatus, getError } from '../helpers/api';
-import { conf } from '../helpers/config';
+import { CALL_API } from '../middleware/call-api';
 import { fetchUserSnaps } from './snaps.js';
 
-const BASE_URL = conf.get('BASE_URL');
-
 export const REPOSITORIES_REQUEST = 'REPOSITORIES_REQUEST';
 export const REPOSITORIES_SUCCESS = 'REPOSITORIES_SUCCESS';
 export const REPOSITORIES_FAILURE = 'REPOSITORIES_FAILURE';
 
-
-export function fetchRepositoriesSuccess(result) {
-  return {
-    type: REPOSITORIES_SUCCESS,
-    payload: {
-      ...result.payload,
-      pageLinks: result.pageLinks
-    }
-  };
-}
-
 export function fetchUserRepositories(pageNumber) {
-  return async (dispatch) => {
-    let urlParts = [BASE_URL, 'api/github/repos'];
+  let urlParts = ['/api/github/repos'];
 
-    dispatch({
-      type: REPOSITORIES_REQUEST
-    });
+  if (pageNumber) {
+    urlParts.push('page/' + pageNumber);
```

Comment:
Or maybe - since there are small updated to serverside of this API in this PR, maybe it can be updated to query be consistent with rest of API calls?